torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -419,7 +418,11 @@ def __init__(
         """
         self.onnxfunction = onnxfunction
         self.param_schema = self.onnxfunction.param_schemas()
-        self.op_schema: onnx.defs.OpSchema = self.onnxfunction.op_schema
+        op_schema = self.onnxfunction.op_schema
+        # Both `OnnxFunction` and `TracedOnnxFunction` never return None for `op_schema`.
+        # However their base class would. Hence return type is annotated as Optional[OpSchema].
+        assert op_schema is not None
```

Comment:
cc @justinchuby for thoughts

Reply:
What is the issue?

Reply:
Kinda unsafe assertion.

Meanwhile, reason for the change is I wasn't able to get intellisense with previous code somehow. Pylance probably still thinks `self.op_schema` is `OpSchema | None`.

Reply:
Should we continue and log? It don't expect this to fail but just in case

Reply:
It if fails it will get reported by the inflight diagnostic.
Continuing also will eventually fail since all code expect it to be not None, so better to fail early?

I wonder if there is some way to annotate correctly. Do we really need `Optional` for `op_schema` for `Op`/`OpLike`?

Reply:
Good point. Maybe we don't. We also need better opschema gen support for ops with variadic inputs I suppose. cc @gramalingam